licence-connect-authenticator/src/main/java/de/intension/authentication/authenticators/backup/jpa/entity/LicenceEntity.java

```diff
@@ -0,0 +1,60 @@
+package de.intension.authentication.authenticators.backup.jpa.entity;
+
+import jakarta.persistence.*;
+
+@Entity
+@Table(name = "LICENCE")
+@NamedQueries({
+    @NamedQuery(name = LicenceEntity.GET_LICENCE_BY_HMAC_ID,
+                query = "SELECT w.content FROM LicenceEntity w WHERE w.hmacId = :hmacId"),
+        @NamedQuery(
+                name = LicenceEntity.REMOVE_LICENCE_BY_HMAC_ID,
+                query = "DELETE FROM LicenseEntity w WHERE w.hmacId = :hmacId")
+})
+public class LicenceEntity {
+    public static final String GET_LICENCE_BY_HMAC_ID = "getLicenceByHmacId";
+    public static final String REMOVE_LICENCE_BY_HMAC_ID = "removeLicenceByHmacId";
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "ID")
+    private Integer id;
+
+    @Column(name = "HMAC_ID", nullable = false)
+    private String hmacId;
```

Comment:
```suggestion
    @Id
    @Column(name = "ID")
    private String id;
```
use hmac ID as ID - check existing Keycloak entities for reference